pkg/ebpf/gadgets/http/tracer/bpf/sniffer_strcuts.h

```diff
@@ -0,0 +1,84 @@
+#include "../../../../include/amd64/vmlinux.h"
+
+#ifdef __TARGET_ARCH_x86
+#include "../../../../include/amd64/vmlinux.h"
+#elif defined(__TARGET_ARCH_arm64)
+#include "../../../../include/arm64/vmlinux.h"
+#endif
+
+#include "../../../../include/types.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_endian.h>
+
+#include "../../../../include/mntns_filter.h"
```

Comment:
First include the builtin includes then your includes.
```c
#include <bpf/bpf_helpers.h>
#include <bpf/bpf_endian.h>

#include "../../../../include/types.h"
#include "../../../../include/mntns_filter.h"
```

Reply:
DONE

Reply:
The
```
#include <bpf/bpf_helpers.h>
#include <bpf/bpf_endian.h>
```
need `#include "../../../../include/types.h" ` otherwise you'll get an error